tests/test_client.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+import requests
+
+from tap_pagerduty.streams import is_fatal_code
+
+
+@pytest.mark.parametrize('status_code', [400, 401, 403, 404,
```

Comment:
same comment as other PR (add a test for `Timeout` or `ConnectionError`)

Reply:
@bobzoller addressed with same decorator routing rationale as [#4](https://github.com/goodeggs/tap-wonolo/pull/4)